src/device/device_ledger.cpp

```diff
@@ -1468,8 +1468,8 @@ namespace hw {
         offset = set_command_header(INS_PREFIX_HASH,2,cnt);      
         len = pref_length - pref_offset;
         //options
-        if (len > (BUFFER_SEND_SIZE-7)) {
-          len = BUFFER_SEND_SIZE-7;
+        if (len > (BUFFER_SEND_SIZE-8)) {
+          len = BUFFER_SEND_SIZE-8;
```

Comment:
Would it be even better to use BUFFER_SEND_SIZE-offset-1 (possibly off by one, but you get the idea) ?

Reply:
yes that good idea.
@grydz

Reply:
Done.